force-app/main/default/classes/BotResponse.cls

```diff
@@ -19,15 +19,14 @@ public class BotResponse {
      * Convenience constructor to create a response with a single message
      */
     public BotResponse(BotMessage message) {
-        this.messages = new BotMessage[]{message};
+        this.messages = new List<BotMessage>{message};
     }
 
     /**
      * Convenience constructor to create a response with a single message
      */
     public BotResponse(BotMessage message, Map<String, String> session) {
-        this.messages = new BotMessage[]{message};
+        this.messages = new List<BotMessage>{message};
```

Comment:
Even though technically `BotMessage[]` is the same as List<BotMessage>` we shouldn't change this since it will surprise the user.